extensions/interactions/base_test.py

```diff
@@ -279,59 +282,44 @@ def test_html_field_types_to_rule_specs_mapping_are_valid(self):
                 html_field_types_to_rule_specs_dict.items()):
             self.assertTrue('format' in html_type_dict)
             self.assertTrue(
-                html_type_dict['format'] in allowed_html_type_formats)
+                html_type_dict['format'] in
+                feconf.ALLOWED_HTML_RULE_VARIABLE_FORMATS)
             del html_type_dict['format']
 
+        html_types = []
```

Comment:
Would it make sense to make this a set, and reduce some checks below accordingly?

Reply:
Yeah done.